lib/class-wp-rest-widgets-controller.php

```diff
@@ -769,44 +672,64 @@ public function get_item_schema() {
 					'required'    => true,
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'rendered'      => array(
+					'description' => __( 'HTML representation of the widget.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'rendered_form' => array(
+					'description' => __( 'HTML representation of the widget admin form.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'edit' ),
+					'readonly'    => true,
+				),
+				'instance'      => array(
+					'description' => __( 'Instance settings of the widget, if supported.', 'gutenberg' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'default'     => null,
+				),
+				'form_data'     => array(
+					'description' => __( 'URL-encoded form data from the widget admin form. Used to update a widget that does not support instance. Write only.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array(),
+					'arg_options' => array(
+						'sanitize_callback' => function( $string ) {
+							$array = array();
+							wp_parse_str( $string, $array );
+							return $array;
+						},
+					),
+				),
+				// BEGIN backwards compatibility. TODO: Remove.
 				'widget_class'  => array(
-					'description' => __( 'Class name of the widget implementation.', 'gutenberg' ),
+					'description' => __( 'DEPRECATED. Class name of the widget implementation.', 'gutenberg' ),
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
 				'name'          => array(
-					'description' => __( 'Name of the widget.', 'gutenberg' ),
+					'description' => __( 'DEPRECATED. Name of the widget.', 'gutenberg' ),
 					'type'        => 'string',
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
 				'description'   => array(
-					'description' => __( 'Description of the widget.', 'gutenberg' ),
+					'description' => __( 'DEPRECATED. Description of the widget.', 'gutenberg' ),
```

Comment:
This is fine, as you can get this data from the widget type api!

Reply:
It's exposed in `instance.raw` when the widget opts in.